### PR TITLE
add new task KeepKeep_Hydrated

### PR DESCRIPTION
**Pull request by Natalsa566**
task KeepKeep_Hydrated

### CODE REVIEW

File: src/main/java/org/academy/kata/implementation/Natalsa566/EightImpl.java [hunk 1]
```diff
@@ -4,7 +4,12 @@
 
 public class EightImpl implements Eight {
     public int liters(double time) {
-        return 0;
+        return (int) Math.floor(time * 0.5);
+    }
+    public static void main(String[] args) {
+        System.out.println(new EightImpl().liters(3));
+        System.out.println(new EightImpl().liters(6.7));
+        System.out.println(new EightImpl().liters(11.8));
```
Comment (lhalam) [inline src/main/java/org/academy/kata/implementation/Natalsa566/EightImpl.java:12]: ?? remove main func

Comment (Natalsa566) [reply src/main/java/org/academy/kata/implementation/Natalsa566/EightImpl.java:12]: done

Comment (lhalam) [reply src/main/java/org/academy/kata/implementation/Natalsa566/EightImpl.java:12]: no

File: src/main/java/org/academy/kata/implementation/Natalsa566/SevenImpl.java [hunk 2]
```diff
@@ -6,10 +6,14 @@ public class SevenImpl implements Seven {
     public long newAvg(double[] arr, double navg) {
         return 0;
     }
-
+@Override
```
Comment (lhalam) [inline src/main/java/org/academy/kata/implementation/Natalsa566/SevenImpl.java:9]: remove `@Override`

Comment (Natalsa566) [reply src/main/java/org/academy/kata/implementation/Natalsa566/SevenImpl.java]: removed

**Pull request merged**